packages/shared/src/modules3rdParty/cross-crypto/index.native.js

```
@@ -1,23 +1,24 @@
+/* eslint-disable no-undef */
 // react-native-crypto
 //      react-native-quick-crypto
 // react-native-get-random-values
 //      (react-native-crypto depend on) react-native-randombytes (deprecated)
 console.log('================ cross-crypto (native)');
 
-if (global.crypto && global.crypto.getRandomValues) {
-  delete global.crypto.getRandomValues;
+if (globalThis.crypto && globalThis.crypto.getRandomValues) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)